fastparquet/writer.py

```diff
@@ -477,10 +524,14 @@ def write_column(f, data, selement, compression=None):
         encoding = "PLAIN_DICTIONARY"
     elif str(data.dtype) in ['int8', 'int16', 'uint8', 'uint16']:
         encoding = "RLE"
+    elif str(data.dtype) in ['float32', 'float64'] and quantization_level is not None:
+        encoding = "LOSSY_RICE"
 
     start = f.tell()
-    bdata = definition_data + repetition_data + encode[encoding](
-            data, selement)
+    encoding_args = [data, selement]
+    if encoding == 'LOSSY_RICE':
+        encoding_args.append(quantization_level)
```

Comment:
My intent is to hide everything and just let the user specify a generic level value. In general it is hard to know what the effects of this are for your specific application before hand. Thus a generic level that is simple to tune makes sense to me.